### DISCUSSION TITLE
How to switch on the Http transfer manager to transfer files with the cellular data connection on Android?

### DISCUSSION THREAD

**Discussion by AntonS127** [shinyorg/shiny#1124] [Q&A] [answered]
Working with the Http File transfer manager I found that the HttpTransferManager uses the Wi-Fi connection only.
I tested it on the Huawei Android 9 and Xiaomi Android 11. 
(The application settings allow to use the Wi-Fi and data connectios.)

Is it possible to switch on the cellular data connection for the Http transfer manager?
Has the Shiny.HttpTransferManager settings for switching the connections?
I've explored the Android Shiny.HttpTransferManager sources but didn't find the settings.

Maybe you know how to switch the connections?

**Comment by aritchie** [comment, marked as answer]
https://github.com/shinyorg/shiny/blob/260-dev/src/Shiny.Net.Http/Platforms/Android/HttpTransferManager.cs#L122

HttpTransferRequest.UseMeteredConnection = true

**Reply by AntonS127** [reply]
Thank you!
It's great!
